source/system/structs/VectorList.ooc

```diff
@@ -118,12 +118,12 @@ VectorList: class <T> extends List<T>{
 			}
 		result
 	}
-	sort: override func (greaterThan: Func (T, T) -> Bool) {
+	sort: override func (isLess: Func (T, T) -> Bool) {
 		inOrder := false
 		while (!inOrder) {
 			inOrder = true
 			for (i in 0 .. this count - 1)
-				if (greaterThan(this[i], this[i + 1])) {
+				if (!isLess(this[i], this[i + 1])) {
```

Comment:
1. `vector = {1, 1, 1, 1, 1}`
2. Call `vector sort()`
3. enter `for` loop, this[0] == 1, this[1] == 1
4. check condition  `!(isLess(1, 1))` == `!(1 < 1)` == `!(false)` == `true`
5. condition evaluates to true, swap elements `this[0] <-> this[1]`